src/main/java/uk/gov/hmcts/sscs/service/NotificationService.java

```diff
@@ -19,11 +22,13 @@
 
     private final NotificationClient client;
     private final NotificationFactory factory;
+    private final ReminderService reminderService;
 
     @Autowired
-    public NotificationService(NotificationClient client, NotificationFactory factory) {
+    public NotificationService(NotificationClient client, NotificationFactory factory, ReminderService reminderService) {
         this.factory = factory;
         this.client = client;
+        this.reminderService = reminderService;
     }
 
     public void createAndSendNotification(CcdResponseWrapper responseWrapper) throws NotificationServiceException {
```

Comment:
if the custom exception extends from RunTimeException there is no need to throws exception in the method signature.